_posts/blog/2016-11-16-simpler-scripts-and-config.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Simpler scripts and configuration coming in Accumulo 2.0.0"
+author: Mike Walch
+reviewers: Josh Elser, Chistopher Tubbs
+---
+
+For the upcoming 2.0.0 release, Accumulo's scripts and configuration were refactored (in 
+[ACCUMULO-4490]) to make Accumulo easier to use. While Accumulo's documentation (i.e. the user
```

Comment:
Better to remove the parenthetical and change it to `[were refactored][ACCUMULO-4490]`.